### PR TITLE
fix(replicator): do not work with empty catalog

### PR DESCRIPTION
**Pull request by peterthomassen**

### CODE REVIEW

File: replicator/run.py [hunk 1]
```diff
@@ -112,6 +112,7 @@ def parse(self):
 
             members.add(name)
 
+        assert members
```
Comment (nils-wisiol) [inline replicator/run.py]: empty members will now crash the replicator. Is that intentional?

Comment (peterthomassen) [reply replicator/run.py]: Yes. Before, if `align-catalog-zone` would run at the stack (when starting the docker-compose application), the catalog zone would be removed and recreated at the stack. For a short time, the zone would then exist but be empty; during that time, slaves would start removing all zones.

It is much better to crash and restart the `replicator` container in this case.

Comment (nils-wisiol) [reply replicator/run.py]: Indeed. `assert` is fine or did you want to raise a proper exception?

**Review by nils-wisiol — DISMISSED** [review]
consider changing to a proper exception. otherwise, lgtm

File: replicator/run.py (hunk 1 above)
Comment (peterthomassen) [reply replicator/run.py]: done

**Review by nils-wisiol — APPROVED** [review]

**Pull request merged**